### PR TITLE
Payments GatewayTimeout logs and metrics

### PR DESCRIPTION
**Pull request by nieznanysprawiciel**

### CODE REVIEW

File: core/payment/src/api/debit_notes.rs [hunk 1]
```diff
@@ -193,27 +199,45 @@ async fn send_debit_note(
...
+                debit_note.recipient_id
+            );
+
             ya_net::from(node_id)
                 .to(debit_note.recipient_id)
                 .service(PUBLIC_SERVICE)
                 .call(SendDebitNote(debit_note))
                 .await??;
             dao.mark_received(debit_note_id, node_id).await?;
+
```
Comment (tworec) [inline core/payment/src/api/debit_notes.rs:217]: counter bumpin (line below) should be next to log msg you've added below. And pls also introduce counter for errors

File: core/payment/src/api/debit_notes.rs [hunk 2]
```diff
@@ -331,14 +359,24 @@ async fn accept_debit_note(
         }
         .await
         {
-            Ok(_) => response::ok(Null),
+            Ok(_) => {
+                log::info!("DebitNote [{}] accepted.", path.debit_note_id);
+                response::ok(Null)
+            }
```
Comment (tworec) [inline core/payment/src/api/debit_notes.rs:365]: ditto: move counter here & introduce counting errors as they are divided into `40x` and `50x`

File: core/payment/src/api/invoices.rs [hunk 3]
```diff
@@ -204,14 +217,20 @@ async fn send_invoice(
         }
         .await
         {
-            Ok(_) => response::ok(Null),
+            Ok(_) => {
+                log::info!("Invoice [{}] sent.", path.invoice_id);
+                response::ok(Null)
+            }
```
Comment (tworec) [inline core/payment/src/api/invoices.rs:223]: ditto

File: core/payment/src/api/invoices.rs [hunk 4]
```diff
@@ -372,14 +416,24 @@ async fn accept_invoice(
         }
         .await
         {
-            Ok(_) => response::ok(Null),
+            Ok(_) => {
+                log::info!("Invoice [{}] accepted.", path.invoice_id);
+                response::ok(Null)
+            }
```
Comment (tworec) [inline core/payment/src/api/invoices.rs:422]: ditto

**Review by maaktweluit — APPROVED** [review]
LGTM!

I do like what 2rec is suggesting :)

**Review by tworec — APPROVED** [review]

**Pull request merged**